chapters/ch04.md

```diff
@@ -2,31 +2,49 @@
 
 ->
 
+# Quy Mô Quyết Định Tiến Trình Phát Triển Machine Learning
+
 
 Many of the ideas of deep learning (neural networks) have been around for decades. Why are these ideas taking off now?
 
 ->
 
+Rất nhiều những ý tưởng của học sâu (mạng neural) đã xuất hiện từ hàng thập kỷ trước.  Vậy tại sao tận bây giờ chúng mới cất cánh, mới bùng nổ như vậy?  
+
+
+
 
 Two of the biggest drivers of recent progress have been:
 
 ->
 
+Hai nguyên nhân mang tính quyết định tới những phát triển gần đây là
+
 
 * **Data availability.​** People are now spending more time on digital devices (laptops, mobile devices). Their digital activities generate huge amounts of data that we can feed to our learning algorithms.
 
 ->
 
+**Dữ liệu sẵn có**. Ngày nay,  mọi người dành nhiều thời gian hơn bên những thiết bị số như máy tính xách tay, thiết bị di động, .v.v. Chính những thiết bị số này là nguồn tạo ra lượng dữ liệu cực lớn - dữ liệu vẫn dùng cho những thuật toán học máy ngày nay.  
+
+
+
 
 * **Computational scale.** ​We started just a few years ago to be able to train neural networks that are big enough to take advantage of the huge datasets we now have.
 
-->
+-> 
+
+**Quy mô năng lực tính toán**. Mãi mấy năm gần đây chúng ta mới có thể huấn luyện mạng neural đủ lớn để tận dụng được lượng dữ liệu big data. 
+
+ 
 
 
 In detail, even as you accumulate more data, usually the performance of older learning algorithms, such as logistic regression, “plateaus.” This means its learning curve “flattens out,” and the algorithm stops improving even as you give it more data:
 
 ->
 
+Cho dù có thêm nhiều nhiều dữ liệu nữa, thì hiệu quả của thuật toán học máy thế hệ cũ như hồi quy logistic cũng chẳng mảy may cải thiện. Nghĩa là đồ thị chất lượng học nằm ngang và thuật toán sẽ không cải thiện chất lượng cho dù thêm bao nhiêu dữ liệu. 
```

Comment:
Em nghĩ không nên dùng **chẳng mảy may** trong hành văn học thuật. Có thể dịch  ngắn gọn là:

" Cho dù có thêm nhiều nhiều dữ liệu nữa, thì ~hiệu quả~ hiệu suất của thuật toán học máy thế hệ cũ như hồi quy logistic cũng ~chẳng mảy may~ không thể cải thiện."

Reply:
Từ 'performance' không có nghĩa là 'hiệu suất', 'hiệu suất' là dịch từ 'productivity' ra. Hiệu suất thường dùng để chỉ lượng công việc thực hiện được trong một đơn gị thời gian, nghĩa đó không phù hợp với hoàn cảnh này.

'Hiệu quả' hay 'kết quả' là từ phù hợp hơn trong văn cảnh này.